docs/source/user_guide/12_list_of_environment_variables.rst

```diff
@@ -0,0 +1,184 @@
+List of Env vars
+__________________
+
+
+.. list-table:: User List
+    :widths: 35 100 20
+    :header-rows: 1
+
+    * - Environment Variable Name
+      - Description
+      - Default value
+    * - .. envvar:: RADICAL_BASE
+      - root dir for temporary state files,
+      - `$HOME/.radical/`
+    * - .. envvar:: RADICAL_PILOT_DBURL
+      - MongoDB URL string and mandatory for RP to work.
+      - {NOT_SET}
+    * - .. envvar:: RADICAL_UTILS_NTPHOST
+      - NTP host to be used for profile syncing
+      - `0.pool.ntp.org`
+    * - .. envvar:: RADICAL_PILOT_BULK_CB
+      - Enable bulk callbacks for performance boost. This changes the callback signature.
+      - FALSE
+    * - .. envvar:: RADICAL_PILOT_STRICT_CANCEL
+      - Cancel the tasks immediately. #TODO Didn't understood
+      - {NOT_SET}
+    * - .. envvar:: RADICAL_DEFAULT_LOG_TGT
+      - The log target. #TODO Need more clarification
+      - .
+    * - .. envvar:: RADICAL_DEFAULT_LOG_DIR
+      - The Log dir
+      - $PWD
+    * - .. envvar:: RADICAL_DEFAULT_LOG_LVL
+      - #TODO Description
+      - ERROR
+    * - .. envvar:: RADICAL_DEFAULT_REPORT
+      - #TODO Description
+      - TRUE
+    * - .. envvar:: RADICAL_DEFAULT_REPORT_TGT
+      - #TODO Description
+      - stderr
+    * - .. envvar:: RADICAL_DEFAULT_REPORT_DIR
+      - Directory of Reporter module
+      - $PWD
+    * - .. envvar:: RADICAL_DEFAULT_PROFILE
+      - #TODO Description
+      - TRUE
+    * - .. envvar:: RADICAL_DEFAULT_PROFILE_DIR
+      - #TODO Description
+      - $PWD
+
+.. raw:: html
+
+   <hr>
+
+.. list-table:: Logging-related Env vars.
+    `ru.Logger` instances have a name and a name space.
+    :widths: 35 100 20
+    :header-rows: 1
+
+    * - Environment Variable Name
+      - Description
+      - Default value
+    * - <NS>_LOG_LVL
+      - Used for log level. #TODO Description
```

Comment:
#TODO @mtitov  Need better description here.

Reply:
Logging level ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"] to control the debug output for a corresponding namespace NS, where NS can be applied as for a specific package (e.g., "RADICAL_PILOT_LOG_LVL" or "RADICAL_UTILS_LOG_LVL") or for a whole stack (e.g., "RADICAL_LOG_LVL").